### PR TITLE
[JENKINS-58028] change gmaven to gmavenplus

### PR DESCRIPTION
**Pull request by bitwiseman**
WIP - Purely for experiment

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
Maybe we should actually have a Groovy profile in  Plugin POM @bitwiseman

**Comment by bitwiseman** [comment]
@oleg-nenashev 
Like https://github.com/jenkinsci/plugin-pom/pull/209 ?  Or something more?

**Comment by rbywater** [comment]
@bitwiseman Is this still a WIP or is something you are wanting to get merged in?

**Comment by bitwiseman** [comment]
Mark this as WIP for now.  I'm going to use this PR to test https://github.com/jenkinsci/plugin-pom/pull/209.  

@oleg-nenashev 
Did you see my question above?

**Comment by oleg-nenashev** [comment]
Sorry, missed it.
https://github.com/jenkinsci/plugin-pom/pull/209 looks good to me. IIUC it will make the build to compine ttests by default so that we can remove the step from here entirely

**Comment by bitwiseman** [comment]
@oleg-nenashev  
No, that change is `pluginManagement` still.

However, once 209 goes in this change would become: 
```
          <plugin>
            <groupId>org.codehaus.gmavenplus</groupId>
            <artifactId>gmavenplus-plugin</artifactId>
            <version>1.7.1</version>
          </plugin>
```

**Comment by oleg-nenashev** [comment]
An additional profile in 209 could be a good addition, but no strong need. Just a possible improvement

**Comment by StefanSpieker** [comment]
@bitwiseman Since it is merged now, is it still WIP? I think you could also upgrade it to 1.8.1

File: pom.xml [hunk 1]
```diff
@@ -55,11 +55,18 @@
...
           </plugin>
           <plugin> <!-- TODO otherwise Groovy tests are uncompilable; cf. similar hack in workflow-cps -->
-            <groupId>org.codehaus.gmaven</groupId>
-            <artifactId>gmaven-plugin</artifactId>
-            <configuration>
-                <providerSelection>1.7</providerSelection>
-            </configuration>
+            <groupId>org.codehaus.gmavenplus</groupId>
+            <artifactId>gmavenplus-plugin</artifactId>
+            <version>1.7.1</version>
```
Comment (bitwiseman) [inline pom.xml:60]:
```suggestion
            <version>1.8.1</version>
```

**Comment by bitwiseman** [comment]
@StefanSpieker 
Updated to 1.8.1.

**Pull request merged**